src/src/WebServer/DevicesPage.cpp

```diff
@@ -1352,6 +1352,11 @@ void devicePage_show_task_values(taskIndex_t taskIndex, deviceIndex_t DeviceInde
     }
 
     LoadTaskSettings(taskIndex);
+ 
+    //placeholder header
+    html_table_header(F(""));
+    ++colCount;
```

Comment:
Maybe you should press the "merge mega branch" button on GitHub (or I can do it for you)
Then you need to fetch and pull the origin branch first before you continue.
Thing is, the DevicesPage has been changed since, so this will cause a merge conflict.